docs/user-guide/rules.md

```diff
@@ -5,7 +5,11 @@ There are over one hundred built-in rules to help you:
 - [avoid errors](#avoid-errors)
 - [enforce conventions](#enforce-conventions)
 
-Many of them are turned on in our [recommended](https://www.npmjs.com/package/stylelint-config-recommended) (Ⓡ) and [standard](https://www.npmjs.com/package/stylelint-config-standard) (Ⓢ) configs.
+Many of them are turned on in our [recommended](https://www.npmjs.com/package/stylelint-config-recommended) and [standard](https://www.npmjs.com/package/stylelint-config-standard) configs.
+
+- ⭐ - [Recommended](https://www.npmjs.com/package/stylelint-config-recommended)
```

Comment:
`⭐` is rendered differently.

Safari :

<img width="231" alt="Screenshot 2023-06-19 at 23 07 55" src="https://github.com/stylelint/stylelint/assets/11521496/ffee04d5-a7f7-4ff5-bafd-510a7c4d8523">

Chrome :

<img width="195" alt="Screenshot 2023-06-19 at 23 08 08" src="https://github.com/stylelint/stylelint/assets/11521496/854aef22-e7b7-4b4f-845a-612953ce27bf">

Firefox : 

<img width="178" alt="Screenshot 2023-06-19 at 23 08 31" src="https://github.com/stylelint/stylelint/assets/11521496/2acea10d-edfc-40ed-bcaa-b7f0f0f15137">

----

This should work everywhere : `&#x2b50;&#xFE0F;`

copy paste this one : ⭐️

Reply:
Thank you! I missed it. Fixed via d2b54e3e.